build/patches/Timezone-Override.patch

```diff
@@ -0,0 +1,1379 @@
++     found in the LICENSE file. -->
++
++<!-- Layout used by the TimezoneOverrideSettingsPreference. -->
++
++<LinearLayout
++    xmlns:android="http://schemas.android.com/apk/res/android"
++    xmlns:app="http://schemas.android.com/apk/res-auto"
++    android:layout_width="match_parent"
++    android:layout_height="wrap_content"
++    android:focusable="false"
++    android:orientation="vertical">
++
++  <org.chromium.components.browser_ui.widget.RadioButtonWithDescriptionLayout
++      android:id="@+id/radio_button_layout"
++      android:layout_width="match_parent"
++      android:layout_height="match_parent">
++
++    <org.chromium.components.browser_ui.widget.RadioButtonWithDescription
++        android:id="@+id/allowed"
++        android:layout_width="match_parent"
++        android:layout_height="wrap_content"
++        android:paddingStart="?android:attr/listPreferredItemPaddingStart"
++        app:primaryText="@string/website_settings_category_timezone_override_allowed" />
++
++    <org.chromium.components.browser_ui.widget.RadioButtonWithEditText
++        android:id="@+id/ask"
++        android:layout_width="match_parent"
++        android:layout_height="wrap_content"
++        android:paddingStart="?android:attr/listPreferredItemPaddingStart"
++        android:inputType="text"
++        android:hint="@string/website_settings_category_timezone_override_custom_hint"
++        app:descriptionText="@string/website_settings_category_timezone_override_custom" />
++
++    <org.chromium.components.browser_ui.widget.RadioButtonWithDescription
++        android:id="@+id/blocked"
++        android:layout_width="match_parent"
++        android:layout_height="wrap_content"
++        android:paddingStart="?android:attr/listPreferredItemPaddingStart"
++        app:primaryText="@string/website_settings_category_timezone_override_random" />
++
++  </org.chromium.components.browser_ui.widget.RadioButtonWithDescriptionLayout>
++
++  <org.chromium.ui.widget.ButtonCompat
++    android:id="@+id/select_button"
++    style="@style/TextButton"
++    android:layout_width="match_parent"
++    android:layout_height="wrap_content"
++    android:paddingBottom="16dp"
++    android:paddingTop="16dp"
++    android:text="@string/website_settings_select_button"
++    app:verticalInset="0dp" />
++
++</LinearLayout>
+diff --git a/components/browser_ui/site_settings/android/java/res/xml/single_website_preferences.xml b/components/browser_ui/site_settings/android/java/res/xml/single_website_preferences.xml
+--- a/components/browser_ui/site_settings/android/java/res/xml/single_website_preferences.xml
++++ b/components/browser_ui/site_settings/android/java/res/xml/single_website_preferences.xml
+@@ -53,6 +53,8 @@
+         android:key="push_notifications_list" />
+     <ListPreference
+         android:key="javascript_permission_list" />
++  <ListPreference
++      android:key="timezone_override_permission_list" />  <!-- bromite #331 -->
```

Comment:
```suggestion
+      android:key="timezone_override_permission_list" />
```

Reply:
You can use IDE plugins that show commit authors

Reply:
ok, maybe you are right but I use visual studio.
my development environment is windows, connected with samba on a virtual debian server, which although in lan is rather slow in connection, especially on such a big project.
what do you suggest for development?

Reply:
No suggestions; I was talking about a plugin like this: https://marketplace.visualstudio.com/items?itemName=waderyan.gitblame

Reply:
> blame

ah ok, I understand.
I use it on codesearch because in vs it doesn't work, too large project.

https://chromium.googlesource.com/chromium/src/+/lkgr/docs/vscode.md
I decided to try vscode with its ssh link: it works wonders for now and has a lot of extensions...

Reply:
Yes, it will also show you the lines which correspond to `git` changes.